### PR TITLE
games-board/crafty: EAPI 8, use https, minor QA fixes

### PR DESCRIPTION
**Pull request by ulm**

### CODE REVIEW

File: games-board/crafty/crafty-24.1-r2.ebuild [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+EAPI=8
+
 inherit flag-o-matic toolchain-funcs
 
 DESCRIPTION="Bob Hyatt's strong chess engine"
-HOMEPAGE="http://www.craftychess.com/"
-SRC_URI="http://www.craftychess.com/${P}.zip
-	http://www.cis.uab.edu/hyatt/crafty/source/${P}.zip
+HOMEPAGE="https://www.craftychess.com/"
+SRC_URI=" https://www.craftychess.com/downloads/source/${P}.zip
```
Comment (chewi) [inline games-board/crafty/crafty-24.1-r2.ebuild:10]: The certificate for https://www.craftychess.com expired 250 days ago.

**Review by chewi — CHANGES_REQUESTED** [review]

File: games-board/crafty/crafty-24.1-r2.ebuild (hunk 1 above)
Comment (ulm) [reply games-board/crafty/crafty-24.1-r2.ebuild]: Let's use archive.org for now.

File: games-board/crafty/crafty-24.1-r2.ebuild [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+
 inherit flag-o-matic toolchain-funcs
 
 DESCRIPTION="Bob Hyatt's strong chess engine"
-HOMEPAGE="http://www.craftychess.com/"
-SRC_URI="http://www.craftychess.com/${P}.zip
-	http://www.cis.uab.edu/hyatt/crafty/source/${P}.zip
+HOMEPAGE="https://web.archive.org/web/20231106192545/http://craftychess.com/"
+SRC_URI="https://web.archive.org/web/20210304102649/http://www.craftychess.com/downloads/source/${P}.zip
 	ftp://ftp.cis.uab.edu/pub/hyatt/documentation/${PN}.doc.ascii"
```
Comment (chewi) [inline games-board/crafty/crafty-24.1-r2.ebuild:11]: Turns out this is dead too. `ftp.cis.uab.edu` doesn't even resolve.

**Review by chewi — CHANGES_REQUESTED** [review]

File: games-board/crafty/crafty-24.1-r2.ebuild (hunk 2 above)
Comment (ulm) [reply games-board/crafty/crafty-24.1-r2.ebuild:11]: Any suggestion for replacement? The file is on Gentoo mirrors, so it's not really a problem. (We could use `mirror://gentoo/` but that won't change much.)

Comment (ulm) [reply games-board/crafty/crafty-24.1-r2.ebuild]: It see the file on https://distfiles.macports.org/crafty/24.1/ but that's not better than our own mirrors.

**Pull request closed without merging**